dropwizard-zipkin/build.gradle

```diff
@@ -0,0 +1,34 @@
+repositories {
+    maven {
+        url "http://central.maven.org/maven2"
+    }
+    // TODO remove these when smartthings-brave is OSS
+    maven {
+        credentials {
+            username smartThingsUserName
+            password smartThingsPassword
+        }
+        url "https://smartthings.artifactoryonline.com/smartthings/libs-release-local"
+    }
```

Comment:
We will need to publish the libraries to Bintray or someplace OSS friendly.  I will work on a solution for this.